.ruby-style.yml

```diff
@@ -233,6 +233,9 @@ Style/SpecialGlobalVars:
   Description: Avoid Perl-style global variables.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#no-cryptic-perlisms
   Enabled: false
+Style/SymbolArray:
+  Description: Check for array literals made up of symbols that are not using the %i() syntax
+  Enabled: false
```

Comment:
I like that rule, but that is a matter of personal preference. I don't mind disabling it. Maybe if we dig a bit deeper to see if there is a technical advantage to use the `%i()` syntax? Or not :P

Reply:
Personnaly, I really like literals. I think the only advantage is that the are shorter to write. 

The rubobop auto correct function should catch them all 🙂